cmd/image-builder/main.go

```diff
@@ -83,6 +81,36 @@ func main() {
 		},
 		TracingConfig: sinit.TracingConfig{
 			Enabled: true,
+			ContextCallback: func(ctx context.Context, attrs []slog.Attr) ([]slog.Attr, error) {
+				if conf.FedoraAuth {
+					val := ctx.Value(fedora_identity.IDHeaderKey)
+					if val == nil {
+						return attrs, nil
+					}
+					id, ok := val.(*fedora_identity.Identity)
+					if !ok {
+						return attrs, nil
+					}
+
+					if id.User != "" {
+						attrs = append(attrs, slog.String("user", id.User))
+					}
+				} else {
+					id, ok := identity.Get(ctx)
+					if !ok {
+						return attrs, nil
+					}
+
+					if id.Identity.OrgID != "" {
+						attrs = append(attrs, slog.String("org_id", id.Identity.OrgID))
+					}
+					if id.Identity.User.Username != "" {
+						attrs = append(attrs, slog.String("user", id.Identity.User.Username))
+					}
+				}
+
+				return attrs, nil
+			},
```

Comment:
This pulls out username/org_id from the context, data is inserted by identity extractors.